### DISCUSSION TITLE
Differences in simulated forces for same flexible rotor (IEA22MW) with different OpenFAST versions

### DISCUSSION THREAD

**Discussion by tdammann-tud** [OpenFAST/openfast#2286] [General]
Dear all,

we simulated the rotor of the IEA 22MW reference turbine with OpenFAST versions 3.4.1 and 2.2.0 in steady inflow conditions (8 ms). When simulating a completely rigid rotor (all DoF's in ElastoDyn off), we obtained the same results for both versions. However, when changing to a flexible rotor (FlapDOF1, FlapDOF2, EdgeDOF on), we saw differences in the spanwise output channels which also lead to differences in the overall Thrust / Torque / ... 
I attached some representative plots to showcase the problem.  Sorry for the missing legend on the graphs, The blue line is version 3.4.1 and the orange line is version 2.2.0
![RotTorque](https://github.com/OpenFAST/openfast/assets/148550445/17389775-01a7-4afd-ba0f-71a5a0088c94)
![spanwise_axInd](https://github.com/OpenFAST/openfast/assets/148550445/f67f9dbb-8fba-41c9-9a9d-dbd2a91861bf)
![spanwise_B1VUndx](https://github.com/OpenFAST/openfast/assets/148550445/cccb1c44-d9cb-428a-a852-1505a26393b9)
![spanwise_Ft](https://github.com/OpenFAST/openfast/assets/148550445/345d179b-604d-4565-8628-b919be0faf97)
![Thrust](https://github.com/OpenFAST/openfast/assets/148550445/a228ba81-7d89-495c-a893-a4894413863c)

Can someone explain why we are obtaining these results?

Best regards,

**Comment by jjonkman** [comment]
Dear @tdammann-tud,

The OpenFAST code changed a lot between these versions (bug fixes, feature enhancements) and it is difficult to identify which specific changes are influencing your specific case.  If you really want to figure that out, I would suggest stepping through the results and comparing release-by-release.  An overview of the changes with each release of OpenFAST are documented: https://github.com/OpenFAST/openfast/releases.  Hopefully the results improve with each release, but certainly bugs are introduced from time to time and fixed later.

You imply that you are using ElastoDyn to model the blade structural dynamics, but the default OpenFAST model of the IEA Wind 22-MW reference wind turbine as supplied on GitHub (https://github.com/IEAWindTask37/IEA-22-280-RWT) relies on BeamDyn.  If you are in fact using BeamDyn, I would encourage you to upgrade to OpenFAST v3.5.3, which is known to resolve previous issues.  (Upgrading may be wise regardless.)

Best regards,
